server/src/models/ip/ip.dto.ts

```diff
@@ -0,0 +1,21 @@
+import {
+  IsNotEmpty,
+  IsString,
+  IsIP,
+} from 'class-validator';
+
+export class AddIPDto {
+  @IsIP()
+  @IsNotEmpty()
+  ip: string;
+
+  @IsNotEmpty()
+  @IsString()
+  description: string;
+}
+
+export class UpdateIPDescriptionDto {
+  @IsNotEmpty()
+  @IsString()
+  description: string;
+}
```

Comment:
If we are giving the user to update the description, would it make sense to give them the ability to update the whitelist as well (so if there is a mistake/update, they do not need to create a new entry).

Reply:
I was actually thinking about that too. My idea for the flow was if the IP was incorrect, a new entry was needed and the incorrect one would need to be deleted.

I can make a quick edit and test out that flow instead, but we can talk about this more

Reply:
mongo atlas allowlist lets you edit the IP so ill probably change IP to be included in the update